### PR TITLE
docs: Confluent Cloud configuration links

### PR DESCRIPTION
**Pull request by astubbs**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -542,6 +542,14 @@ image::https://lucid.app/publicSegments/view/43f2740c-2a7f-4b7f-909e-434a5bbe3fb
 
 See the link:{project_root}/parallel-consumer-examples/parallel-consumer-example-streams/src/main/java/io/confluent/parallelconsumer/examples/streams/StreamsApp.java[Kafka Streams example] project, and it's test.
 
+[[confluent-cloud]]
+=== Confluent Cloud
+
+. After https://kafka-tutorials.confluent.io/creating-first-apache-kafka-producer-application/confluent.html#provision-your-fully-managed-kafka-cluster-in-confluent-cloud[provisioning a Confluent Cloud cluster].
```
Comment (ybyzek) [inline README.adoc:548]: @astubbs There's an open PR to Kafka Tutorials to change the name of this step's heading, so this link will break.  To future proof, I think it's better NOT to link to Kafka Tutorials but rather pull the text into here.

Comment (astubbs) [reply README.adoc]: ack. updated - check it out :)

**Comment by astubbs** [comment]
@ybyzek FYI rendered view here: [diff]( https://github.com/confluentinc/parallel-consumer/diffs/1?base_sha=c1dac3859d9a96562e7f1e75540be301f3252c0f&commentable=true&head_user=astubbs&pull_number=167&sha1=c1dac3859d9a96562e7f1e75540be301f3252c0f&sha2=9af1d79eb46179654782cdd72990750836054998&short_path=1194561&unchanged=expanded&w=false#confluent-cloud) and [raw](https://github.com/astubbs/parallel-consumer/tree/cloud-docs#confluent-cloud)

File: README.adoc [hunk 2]
```diff
@@ -542,6 +543,22 @@ image::https://lucid.app/publicSegments/view/43f2740c-2a7f-4b7f-909e-434a5bbe3fb
...
 See the link:{project_root}/parallel-consumer-examples/parallel-consumer-example-streams/src/main/java/io/confluent/parallelconsumer/examples/streams/StreamsApp.java[Kafka Streams example] project, and it's test.
 
+[[confluent-cloud]]
+=== Confluent Cloud
+
+. Provision your fully managed Kafka cluster in Confluent Cloud
+.. Sign up for https://www.confluent.io/confluent-cloud/tryfree/[Confluent Cloud], a fully-managed Apache Kafka service.
+.. After you log in to Confluent Cloud, click on `Add cloud environment` and name the environment `learn-kafka`.
+Using a new environment keeps your learning resources separate from your other Confluent Cloud resources.
+.. From the `Billing & payment` section in the Menu, apply the promo code `CC100KTS` to receive an additional $100 free usage on Confluent Cloud (https://www.confluent.io/confluent-cloud-promo-disclaimer[details]).
```
Comment (ybyzek) [inline README.adoc:553]: `CC100KTS` is specifically for Kafka Tutorials tracking and attribution so it should not be used here.  You could create a unique Confluent Cloud promo code for this project, or just delete this line.

File: README.adoc [hunk 3]
```diff
@@ -542,6 +543,22 @@ image::https://lucid.app/publicSegments/view/43f2740c-2a7f-4b7f-909e-434a5bbe3fb
...
+.. Click on https://confluent.cloud/learn[LEARN] and follow the instructions to launch a Kafka cluster and to enable Schema Registry.
+. Access the client configuration settings
+.. From the Confluent Cloud Console, navigate to your Kafka cluster.
+From the `Clients` view, get the connection information customized to your cluster (select `Java`).
+.. Create new credentials for your Kafka cluster, and then Confluent Cloud will show a configuration block with your new credentials automatically populated (make sure `show API keys` is checked).
+.. Use these settings presented to https://docs.confluent.io/clients-kafka-java/current/overview.html[configure your clients].
+. Use these clients for steps outlined in the <<common_preparation>> section.
```
Comment (ybyzek) [inline README.adoc:560]: @astubbs Even from the `common_preparation`, it's not clear which files need to be modified and how the settings should be configured.

**Review by ybyzek — COMMENTED** [review]
Thanks @astubbs , left some comments.

File: README.adoc (hunk 3 above)
Comment (astubbs) [reply README.adoc:560]: There is no file to be modified - the examples just mock out the clients. The step above links to the instructions on how to configure and instantiate the clients. The common prep shows how to pass those clients into PC. Make sense?

File: README.adoc (hunk 2 above)
Comment (astubbs) [reply README.adoc:553]: Deleted

File: README.adoc (hunk 3 above)
Comment (ybyzek) [reply README.adoc:559]: 👍 got it.  Thanks!

**Review by ybyzek — APPROVED** [review]
LGTM @astubbs , thank you!

**Pull request merged**